src/test/java/nextstep/subway/line/application/LineServiceTest.java

```diff
@@ -0,0 +1,176 @@
+package nextstep.subway.line.application;
+
+import nextstep.subway.line.domain.Line;
+import nextstep.subway.line.domain.LineRepository;
+import nextstep.subway.line.dto.LineRequest;
+import nextstep.subway.line.dto.LineResponse;
+import nextstep.subway.line.dto.SectionRequest;
+import nextstep.subway.station.application.StationService;
+import nextstep.subway.station.domain.Station;
+import nextstep.subway.station.dto.StationResponse;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import static nextstep.subway.line.domain.LineFixture.*;
+import static nextstep.subway.station.StationFixture.*;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+public class LineServiceTest {
+
+    @Mock
+    LineRepository lineRepository;
+    @Mock
+    StationService stationService;
+    @InjectMocks
+    LineService lineService;
+
+    private Line 일호선;
+    private Line 이호선;
+    private Line 신이호선;
+    private Station 서울역;
+    private Station 시청역;
+    private Station 남영역;
+
+    @BeforeEach
+    void setup() {
+        일호선 = 일호선();
+        이호선 = 이호선();
+        신이호선 = 신이호선();
+        남영역 = 남영역();
+        서울역 = 서울역();
+        시청역 = 시청역();
+
+    }
+    @DisplayName("노선을 저장한다.")
+    @Test
+    void 노선_저장_테스트() {
+        // given
+        LineRequest lineRequest = new LineRequest("1호선", "Blue", 서울역().getId(), 시청역().getId(), 20);
+
+        given(stationService.findById(서울역.getId())).willReturn(서울역);
+        given(stationService.findById(시청역.getId())).willReturn(시청역);
+        given(lineRepository.save(any())).willReturn(일호선);
+
+        // when
+        LineResponse lineResponse = lineService.saveLine(lineRequest);
+        List<String> stationNames = lineResponse.getStations().stream()
+                .map(StationResponse::getName)
+                .collect(Collectors.toList());
+
+        // then
+        verify(lineRepository, times(1)).save(any());
```

Comment:
```suggestion
        verify(lineRepository).save(any());
```

times(1)은 생략할 수 있습니다.
verify를 사용하셔도 큰 상관은 없으나 BDD 스타일을 더 잘 표현하는 then 을 사용하시는걸 추천드리고 있어요.